web/ui/react-app/src/pages/targets/JobsList.test.tsx

```diff
@@ -19,9 +20,12 @@ describe('Flags', () => {
   });
 
   describe('before data is returned', () => {
-    it('renders a Loader', () => {
+    it('renders a spinner', () => {
       const jobsList = shallow(<JobsList {...defaultProps} />);
-      expect(jobsList.find(Loader)).toHaveLength(1);
+      const spinner = jobsList.find(FontAwesomeIcon);
+      expect(spinner).toHaveLength(1);
+      expect(spinner.prop('icon')).toEqual(faSpinner);
+      expect(spinner.prop('spin')).toBe(true);
```

Comment:
You can use ```toBeTruthy()``` ... Spinner count, I would check in a separate test like -```it('should not render multiple spinners')``` or something like that

Reply:
Again, i prefer `toBe(true)`. But i will split it into multiple tests as you've suggested. Thanks!

Reply:
done

Reply:
Yes, didn't take into account that toBeFalsy/Truthy will handle values different than true/false.